core/docker-entrypoint.sh

```diff
@@ -137,17 +128,55 @@ admin_pass: $MAILMAN_REST_PASSWORD
 class: mailman_hyperkitty.Archiver
 enable: yes
 configuration: /etc/mailman-hyperkitty.cfg
+
 EOF
 
-# Generate a basic configuration to use postfix.
-cat > /etc/postfix-mailman.cfg <<EOF
+# Generate a basic configuration to use exim
+cat > /etc/exim-mailman.cfg <<EOF
+[mta]
+incoming: mailman.mta.exim4.LMTP
+outgoing: mailman.mta.deliver.deliver
+lmtp_host: $MM_HOSTNAME
+lmtp_port: 8024
+smtp_host: $SMTP_HOST
+smtp_port: $SMTP_PORT
+configuration: python:mailman.config.exim4
+
+EOF
+
+cat > /etc/postfix-mailman-configuration.cfg << EOF
```

Comment:
Can we keep this at it's original location `/etc/postfix-mailman.cfg` and change the rest of the temporary files ('/etc/postfix-mailman.cfg` and `/etc/exim-mailman.cfg`) to be in `/tmp` instead of `/etc` please?

Reply:
That is nicer indeed. I updated it in my latest commit.